authzed/api/v1/permission_service.proto

```diff
@@ -406,16 +406,41 @@ message LookupSubjectsResponse {
 
   // subject_object_id is the Object ID of the subject found. May be a `*` if
   // a wildcard was found.
-  string subject_object_id = 2;
+  // deprecated: use `subject`
+  string subject_object_id = 2 [deprecated = true];
 
   // excluded_subject_ids are the Object IDs of the subjects excluded. This list
   // will only contain object IDs if `subject_object_id` is a wildcard (`*`) and
   // will only be populated if exclusions exist from the wildcard.
-  repeated string excluded_subject_ids = 3;
+  // deprecated: use `excluded_subjects`
+  repeated string excluded_subject_ids = 3 [deprecated = true];
 
   // permissionship indicates whether the response was partially evaluated or not
-  LookupPermissionship permissionship = 4 [ (validate.rules).enum = {defined_only: true, not_in: [0]} ];
+  // deprecated: use `subject.permissionship`
+  LookupPermissionship permissionship = 4 [ deprecated = true, (validate.rules).enum = {defined_only: true, not_in: [0]} ];
 
   // partial_caveat_info holds information of a partially-evaluated caveated response
-  PartialCaveatInfo partial_caveat_info = 5 [ (validate.rules).message.required = false ];
+  // deprecated: use `subject.partial_caveat_info`
+  PartialCaveatInfo partial_caveat_info = 5 [ deprecated = true, (validate.rules).message.required = false ];
```

Comment:
Since this field was only ever used in an experiment, can we make a breaking change here?

Reply:
I did that originally but @vroldanbet pointed out that it would be a breakage change that is in a release, so I changed it back

Reply:
but it's an experimental feature

Reply:
@jzelinskie even if it's experimental, wouldn't this change break folks using `LookupSubjects` even if they didn't use the new fields? E.g. a client updated to `authzed-go` prior to this change. Wouldn't that client fail to parse the response from the server?

Reply:
Confirmed via the wire format that this is a breaking change, so yeah we've got to do this.